src/components/Data/Documents/Page.vue

```diff
@@ -390,6 +405,41 @@ export default {
     }
   },
   methods: {
+    async subscribeToCurrentDocs() {
+      try {
+        if (this.subscribeRoomId) {
+          await this.$kuzzle.realtime.unsubscribe(this.subscribeRoomId)
+          this.subscribeRoomId = null
+        }
+        const roomId = await this.$kuzzle.realtime.subscribe(
+          this.indexName,
+          this.collectionName,
+          {
+            or: this.documents.map(d => ({ equals: { _id: d._id } }))
+          },
+          this.realtimeNotifCallback
+        )
+        this.subscribeRoomId = roomId
+      } catch (error) {
+        this.$log.error(error)
+      }
+    },
+    realtimeNotifCallback(notif) {
+      if (notif.scope === 'out') {
+        return
```

Comment:
Should delete the document from the list, shouldn't it? Or, at least, mark the document as "DELETED" in the list (so that the user doesn't see stuff disappear under their eyes...)

Reply:
Same as above